content/source/docs/enterprise/users-teams-organizations/users.html.md

```diff
@@ -36,6 +36,10 @@ To add a user to an organization, a member of that organization's owners team mu
 
 Adding a user to a team requires only their username.
 
+## Site Admin Permissions
+
+On private installs of TFE, some user accounts have a special "site admin" permission that allows administration of the entire instance. For more information, see [Administering Private Terraform Enterprise](../private/admin/index.html).
```

Comment:
I'd prefer we use "installations" or "instances" or "On Private TFE installations".

I'd also love to add a note here that site admin permissions are disjoint from organization membership permissions; admins have admin permission for everything, but regular user type permission only for those resources they have ordinary access to. This question commonly comes up with API token permissions, but it would be nice to clarify here generally.

Reply:
Good point.